src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/PrintReadsSparkIntegrationTest.java

```diff
@@ -3,22 +3,50 @@
 import org.broadinstitute.hellbender.CommandLineProgramTest;
 import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
 import org.broadinstitute.hellbender.exceptions.UserException;
-import org.broadinstitute.hellbender.utils.test.IntegrationTestSpec;
-import org.broadinstitute.hellbender.utils.test.SamAssertionUtils;
 import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.test.SamAssertionUtils;
+import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
 import java.io.File;
 import java.io.IOException;
-import java.util.Arrays;
 
 public final class PrintReadsSparkIntegrationTest extends CommandLineProgramTest {
 
+    private static final File TEST_DATA_DIR = getTestDataDir();
+
     @Override
     public String getTestedClassName() {
         return PrintReadsSpark.class.getSimpleName();
     }
 
+    @DataProvider(name="testingDataCRAM")
+    public Object[][] testingDataCRAM() {
+        return new String[][]{
+                {"print_reads.sorted.cram", ".sam", "print_reads.fasta"},
+                {"print_reads.sorted.cram", ".bam", "print_reads.fasta"},
+                {"print_reads.sorted.cram", ".cram", "print_reads.fasta"},
+                {"print_reads.sorted.sam", ".cram", "print_reads.fasta"},
+                {"print_reads.sorted.bam", ".cram", "print_reads.fasta"}
+        };
+    }
+
+    @Test(dataProvider="testingDataCRAM")
+    public void testFileToFileCRAM(String fileIn, String extOut, String reference) throws Exception {
+        String samFile= fileIn;
+        final File outFile = File.createTempFile(samFile + ".", extOut);
```

Comment:
great catch @cmnbroad! it's totally bogus that we claim we're writing a cram file here while in fact it's just a bam file - we need to validate it https://github.com/broadinstitute/gatk/issues/1269